src/kodeagent/agents/csv_agent.py

```
@@ -0,0 +1,596 @@
+                        df = pd.read_csv(file_path)
+                        _agent_df_storage.set(df)
+                        logger.info('Auto-loaded CSV from %s', file_path)
+                        yield self.response(
+                            rtype='log',
+                            value=(
+                                f'Auto-loaded dataset from {file_path}. Rows: {df.shape[0]},'
+                                f' Cols: {df.shape[1]}'
+                            ),
+                            channel='run',
+                        )
+                    except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)